bin/lib/recreate_database.py

```diff
@@ -14,11 +16,11 @@ def main():
     print '****************************************************'
     print
     while True:
-        response = raw_input('Proceed [y/n]? ')
-        if response.lower() == 'y':
+        response = raw_input('Proceed [yes/no]? ')
```

Comment:
The fact that users are used to typing "y" is the reason I chose to make it longer. Dropping a database accidentally is about the worst thing I can think of happening.